packages/aws-rfdk/lib/lambdas/nodejs/configure-spot-event-plugin/test/handler.test.ts

```diff
@@ -418,6 +438,15 @@ describe('SEPConfiguratorResource', () => {
       const mockConfigureSpotEventPlugin = jest.fn( (a) => returnTrue(a) );
       mockSpotEventPluginClient.configureSpotEventPlugin = mockConfigureSpotEventPlugin;
 
+      async function returnEmptyArray(_v1: any): Promise<string[]> {
+        return [];
+      }
+      const mockGetGroupPoolCollection = jest.fn( (a) => returnEmptyArray(a) );
```

Comment:
nit: Could we do this instead of defining an inner function?
```suggestion
      const mockGetGroupPoolCollection = jest.fn( async (_a) => [] );
```

I see that we're already using inner functions elsewhere in this file, so I'm okay with leaving this as is.